plugin/ocgrpc/client_stats_handler_test.go

```diff
@@ -318,6 +319,16 @@ func TestClientDefaultCollections(t *testing.T) {
 				t.Errorf("%q: RetrieveData(%q) = %v", tc.label, wantData.v().Name, err)
 				continue
 			}
+			for i := range gotRows {
+				switch data := gotRows[i].Data.(type) {
+				case *view.CountData:
+					data.Start = time.Time{}
```

Comment:
Can you please leave a comment about why do we need to overwrite the `Start`?
I believe it's needed to be able to compare "want" and "got" (otherwise they will be considered non-equal).

Also, it seems that you have more than one place doing this, so maybe extract it to an utility function `func ResetStart(data *view.CountData)`?

Reply:
I wasn't sure what package to add such a test helper to so I just added the helper to each package that needed it.

Reply:
I went ahead and moved this to aggregation_data.go in the view package. Let me know if this is not the right place to put it.